examples/usecases/entertainment-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<p>You should be able to read the information <code>1682</code> (how many movieIds are avalable) from the schema file.</p><p> </p><p><code>train.schema</code> </p><p> </p><p>Then we do not need to read the output of the NVTabular workflow. This could create issues when paths are not consistent with preprocessing and where the notebook is stored.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Good point on using paths!</p><p> </p><p>Here I don't seem to be able to find this information in the schema. It only contains info on cardinality but not the maximum possible value.</p><p> </p><p>I switched to grabbing the path from the schema, please let me know if that would be okay</p><p> </p><p><code></code><code>movieIds = cudf.read_parquet(train.schema.column_schemas['movieId'].properties['cat_path'])</code><code></code></p>

Reply:
saw your comment below, switched to synthetic data where this no longer is an issue 🙂